src/features/checkout/screen/formInput.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
I'm curious why the file extension is `.jsx` rather than `.tsx`, is there a particular reason for this?